spec/test_app/app/controllers/basechurch/v1/announcements_controller_spec.rb

```diff
@@ -63,7 +74,7 @@
 
       context 'with invalid parameters' do
         let(:invalid_params) do
-          post_params[:announcements][:description] = ''
+          post_params[:data][:attributes][:description] = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.